lib/dice/src/handoff.rs

```diff
@@ -0,0 +1,73 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::{AliasCert, AliasOkm, DeviceIdCert};
+use hubpack::SerializedSize;
+use lpc55_pac::syscon::RegisterBlock;
+use serde::{Deserialize, Serialize};
+
+// This memory is the USB peripheral SRAM that's 0x4000 bytes long. Changes
+// to this address must be coordinated with the [dice_*] tables in
+// chips/lpc55/chips.toml
```

Comment:
build.rs could generate these from the config.

Reply:
I spent a bit of time looking at this and while I agree in general it would require adding some plumbing to xtask. The data we would need is in the 'chip' key from the top level toml table but AFAIK the currently available methods for getting at the app toml don't expose it:
- `build_util::config` exposes the app config that includes tables under `[config]`
- `build_util::task_config` exposes tables under the `[task_name]` table
So that's not to say it can't be done, but AFAIK doing so would require a new interface in build_util and some plumbing in xtask to expose the chip key / file path. This is definitely something that should be added but maybe it can wait till after this PR is merged? https://github.com/oxidecomputer/hubris/issues/731